packages/layer-selector/LayerSelector.js

```diff
@@ -265,6 +264,7 @@ const LayerSelector = (props) => {
 
       setManagedLayers(managedLayersDraft);
     });
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
these deps should be updated or no?

Reply:
No, I tried and it gets stuck in a loop.